cmd/bee/cmd/cmd.go

```diff
@@ -238,6 +239,7 @@ func (c *command) setAllFlags(cmd *cobra.Command) {
 	cmd.Flags().String(optionNamePostageContractAddress, "", "postage stamp contract address")
 	cmd.Flags().String(optionNamePriceOracleAddress, "", "price oracle contract address")
 	cmd.Flags().String(optionNameTransactionHash, "", "proof-of-identity transaction hash")
```

Comment:
Somehow mention here or in a comment elsewhere that the order of precedence is to use the tx-hash that is already stored, then the tx-hash that is passed in and then the tx-hash that is acquired through deploying a new chequebook.